client/components/search/index.jsx

```diff
@@ -266,10 +267,13 @@ const Search = React.createClass( {
 				this.props.initialValue;
 
 		const autocorrect = this.props.disableAutocorrect && {
-			autoComplete: 'off',
-			autoCorrect: 'off',
-			spellCheck: 'false'
-		};
+				autoComplete: 'off',
+				autoCorrect: 'off',
+				spellCheck: 'false'
+			},
+			maxLength = this.props.maxLength && {
```

Comment:
Not really. I couldn't make it work down there. :) And I want it to be conditional.